src/adapter/debugAdapter.ts

```diff
@@ -322,6 +326,17 @@ export class DebugAdapter {
     return {};
   }
 
+  async _toggleSkipFileStatus(
+    params: Dap.ToggleSkipFileStatusParams,
+  ): Promise<Dap.ToggleSkipFileStatusResult | Dap.Error> {
+    if (!this._thread?.pausedDetails()) {
+      return errors.createSilentError(localize('error.threadNotPaused', 'Thread is not paused'));
```

Comment:
Does it need to be paused? Could I toggle skipping for a file even before I hit a BP?

Reply:
Actually, that's a good point. I put that in because of [this](https://github.com/microsoft/vscode-js-debug/pull/94#discussion_r350502705) comment, but I'm not sure why it needs to be paused. @roblourens ?

Reply:
Yeah I guess that's not required, the command is still in the command palette